### PR TITLE
Routes, content, and other tweaks

### PR DESCRIPTION
**Pull request by batemapf**
## Description
Included in this PR: 
- [x] New GA prefix `covid-vaccination-expanded-`; [have asked about](https://dsva.slack.com/archives/C1Q4294JJ/p1616904711023500) getting it added to our GA property for tracking
- [x] Lots of content updates per @DanielleSoCompany's [wonderful content doc](https://github.com/department-of-veterans-affairs/vets-website/files/6216859/covid.form_latest.content.changes_03.26.21.docx)
- [x] Set `submitUrl` to new vets-api endpoint
- [x] Disabled prefill since that is not expected behavior and I don't want it to mess up the experience should someone enter the form logged in
- [x] Overhauled the route structure in hopes of achieving the following:
- Vaccine mini-hub: `/health-care/covid-19-vaccine` (no change)
- Sign up for a vaccine introduction page: `/health-care/covid-19-vaccine/sign-up/stay-informed` (still/formerly the KMI intro page; this is the most awkward route IMO; if we want to tell people to go straight to the tool, will likely want to make this better)
- Health care status selection page: `/health-care/covid-19-vaccine/sign-up`
- KMI form: `/health-care/covid-19-vaccine/sign-up/stay-informed/form` (this is long)
- First page of expanded form: `health-care/covid-19-vaccine/sign-up/eligibility`
- Second page of expanded form: `health-care/covid-19-vaccine/sign-up/veteran-information`
- etcetera...

## Testing done
Local testing in browser using staging-api.va.gov.

## Screenshots
Does not include eligibility selection screen, as there is work happening on that over in #16544.
Does not include veteran service history screen.

Screenshot_2021-03-27 Sign up to get a COVID-19 vaccine at VA
Screenshot_2021-03-27 Sign up to get a vaccine
Screenshot_2021-03-27 Sign up to get a vaccine(1)
Screenshot_2021-03-27 Sign up to get a vaccine(2)
Screenshot_2021-03-28 Sign up to get a vaccine(1)
Screenshot_2021-03-28 Sign up to get a vaccine

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by batemapf** [comment]
@DanielleThierryUSDSVA see above for latest screens ☝️

File: src/applications/coronavirus-vaccination-expansion/config/form.js [hunk 1]
```diff
@@ -20,38 +21,33 @@ import manifest from '../manifest.json';
 const formConfig = {
   rootUrl: manifest.rootUrl,
   urlPrefix: '/',
-  // submitUrl: '/v0/api',
+  submitUrl: `${environment.API_URL}/covid_vaccine/v0/expanded_registration`,
   submit: () =>
```
Comment (patrickvinograd) [inline src/applications/coronavirus-vaccination-expansion/config/form.js:25]: I believe you need to remove this `submit` function on line 25 as it is overriding the default submit behavior and just returning a mocked response.

**Review by theodur — APPROVED** [review]
Everything LGTM from a platform perspective 👍. The `yarn.lock` file looks like it may need to be updated.

**Review by stephenBarrs — APPROVED** [review]
LGTM!

**Pull request merged**